templates/burp.conf.j2

```diff
@@ -24,8 +24,17 @@ ssl_peer_cn = {{ burp_client_ssl_peer_cn }}
 {% for l in burp_client_ports_per_operation %}
 {{ l }}
 {% endfor %}
+{% endif %}
 
+{% if burp_client_backup_script is defined %}
```

Comment:
Probably should be burp_client_backup_script_bool with false by default?

Reply:
Yes! sounds sensible.
However please note I'm having issues using ZFS backup over FIFO with this script. It's a long time ago would need to look into this to be more precise. Maybe not of use for you.